libs/azure/tests/test_azure.py

```diff
@@ -0,0 +1,38 @@
+"""Test thredds downloader code
+"""
+from odc.azure import find_blobs, download_yamls
+
+
+def test_find_blobs():
+    """Find blobs in a sample Azure account, this will fail if the blob store changes or is removed
+    """
+
+    account_name = "geoau"
+    account_url = "https://" + account_name + ".blob.core.windows.net"
+    container_name = "ey-gsa"
+    credential = "sv=2019-10-10&si=ey-gsa-ro&sr=c&sig=4T2mncGZ2v%2FqVsjjyHp%2Fv7BZytih8b251pSW0QelT98%3D"
```

Comment:
is that a secret secret?

Reply:
i guess it is - it is read only, and we were planning on distributing it with the EY datascience challenge, but the issue is we dont currently have a good location that we control. I was tempted to use my personal Azure, but that isn't going to available once my trail runs out and I am not aware of any other location in Azure where yamls are stored.

Reply:
Best to read from `os.environ` and either default to that one, or skip test if not set.